README.md

```diff
@@ -76,6 +76,8 @@ Persistent settings
 
 ### SSH Keys
 
+Generating a pair of SSH keys allow you to ssh login to your server without typing or remembering your password. There is a private and public key. You keep your private key secret and safe and give out your public key to any server you would like to securely connect to. SSH keys are better than passwords because you cannot "forget" them and they are not easily figured out or comprimised.
```

Comment:
compromised spelling change

Reply:
thank you for your review but maybe you should fix it yourself next time @racheljoyrodas .